### PR TITLE
Server Refactoring (RIPD-1120)

### PR DESCRIPTION
**Pull request by vinniefalco**
This is a purely mechanical refactoring

### CODE REVIEW

**Comment by miguelportilla** [comment]
OS X compiler errors, will PM.

File: src/ripple/rpc/impl/ServerHandlerImp.h [hunk 1]
```diff
@@ -42,69 +40,42 @@ bool operator< (Port const& lhs, Port const& rhs)
     return lhs.name < rhs.name;
 }
 
-class WSInfoSub : public InfoSub
+class ServerHandlerImp
+    : public Stoppable
```
Comment (ximinez) [inline src/ripple/rpc/impl/ServerHandlerImp.h]: Devirtualizing: good. Keeping the `Imp` suffix: meh.

Comment (vinniefalco) [inline src/ripple/rpc/impl/ServerHandlerImp.h]: Yeah...well:
```
using ServerHandler = ServerHandlerImp;
```
This is just a down payment on a bigger refactoring. Getting these changes in will mean fewer conflicts later. Its a mechanical refactoring (no logic changes).

Comment (ximinez) [inline src/ripple/rpc/impl/ServerHandlerImp.h]: I did see that alias, and now that I understand the reasoning, it makes sense.

**Comment by ximinez** [comment]
Looks good. :+1:

**Comment by seelabs** [comment]
:+1:

**Pull request closed without merging**